packages/conf-librsvg2/conf-librsvg2.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "The gtksourceview programmers"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://wiki.gnome.org/Projects/LibRsvg"
+license: "LGPL-2.1-or-later"
+build: [["pkg-config" "--short-errors" "--print-errors" "librsvg-2.0"]]
```

Comment:
```suggestion
build: [["pkg-config" "--short-errors" "--print-errors" "librsvg >= 2.0"]]
```
Shouldn't it be checked with the comparison?

Reply:
No, pkg-config considers that the name of the package is librsvg-2.0. The -2.0 is not a version constraint, it's just part of the package name. 
```
$ pkg-config --short-errors --print-errors "librsvg >= 2.0"
No package 'librsvg' found
```

Reply:
I see, thanks!